internal/vcs/create-git-repos.go

```diff
@@ -5,34 +5,71 @@ import (
 	"fmt"
 	"github.com/machinebox/graphql"
 	"os/exec"
+	"strings"
 )
 
-// this is being developed with the following assumptions:
-// 1. create.go will handle parsing the repo names and github credentials and pass them into initializeRepostiories().
-// 2. if organizationName is set, create an org owned repo.  if not, create a personal repo.
+// InitializeRepository Creates and initializes a github repository for the given url
+// repositoryUrl is expected to be in the format "github.com/{ownerName}/{repositoryName}"
+func InitializeRepository(repositoryUrl string, githubApiKey string) {
 
-// takes a list of subdirectories containing modules to create a repository and do initial commit for
-func InitializeRepositories(moduleDirs []string, remoteRepository string, organizationName string, githubApiKey string) {
+	ownerName, repositoryName, pErr := parseRepositoryUrl(repositoryUrl)
+	if pErr != nil {
```

Comment:
Any reason to not just have `err` and reuse it?

Reply:
I like to keep errors scoped as close to the call as possible, but its not really needed here.  I can change this to reuse a single err.